libs/ts-rest/core/src/lib/client.ts

```diff
@@ -188,6 +169,7 @@ export type ApiFetcherArgs = {
   rawQuery: unknown;
   contentType: AppRouteMutation['contentType'];
   credentials?: RequestCredentials;
+  signal?: AbortSignal;
```

Comment:
Hmmm, is there a way to make this only part of the `initQueryClient` in the `@ts-rest/react-query` package? 
To me, it doesn't make sense to add this property/typing for the core package, since this is react-query specific?

Reply:
It's actually a standard web API, you can use an AbortController to cancel any fetch request https://developer.mozilla.org/en-US/docs/Web/API/AbortController#examples

So it may be useful if you want to cancel large downloads/uploads, etc.

Reply:
Oh sweet! ship it!